src/registry_schemas/example_data/schema_data.py

```diff
@@ -85,6 +85,165 @@
     'deliveryInstructions': 'some delivery instructions'
 }
 
+COURT_ORDER = {
+    'fileNumber': '#1234-5678/90',
+    'orderDate': '2021-01-30T09:56:01+08:00',
+    'effectOfOrder': 'planOfArrangement',
+    'orderDetails': 'A note about order',
+    'fileKey': '011e332d-1b8e-4218-8710-ad8ac1fbc592.pdf'
+}
+
+FOREIGN_JURISDICTION = {
+    'country': 'CA',
+    'region': 'AB'
+}
+
+AMALGAMATION = {
+    'type': 'regular',
+    'amalgamatingBusinesses': [
+        {
+            'role': 'amalgamating',
+            'identifier': 'BC1234567'
+        },
+        {
+            'role': 'amalgamating',
+            'legalName': 'Foreign Co.',
+            'foreignJurisdiction': FOREIGN_JURISDICTION,
+            'corpNumber': '123456'
+        }
```

Comment:
Yes, I think both objects here are correct.

For foreign companies, we store the 3 data fields that staff entered (+ role).

For BC companies, we store just their identifier (+ role) -- we can look up everything else.